### ISSUE TITLE
Pre-Publish Checklist: Warnings Logic

### ISSUE THREAD

**Issue by carloskelly13** [GoogleForCreators/web-stories-wp#4911]
## Feature Description

Implement the logic to throw an error if one or more of the warnings are present:

**Potential accessibility issues (warnings)**

- Low contrast between font and background color
- Font size too small (<12)
- Very low image resolution: actual image asset on screen, at the current zoom, offers <1x pixel density (guideline is to strive for >828 x 1792)
- Video doesn’t include subtitles, captions or title
- Too many links on a page (more than 3)
- Link tappable region too small
- Images missing alt text

**Potential distribution issues (warnings)**

- Missing story description

https://docs.google.com/document/d/1GJZljicdXcP7oRdywsF3EGUaCVpCHsDrK2gUMubnrbk/edit?pli=1#

## Additional Context

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by zachhale** [comment]
@carlos-kelly is story description actually the "excerpt"?
